README.md

```diff
@@ -95,17 +95,17 @@ This enables or disables the provider named `<provider>`. Valid options are:
 
 Consider the following example:
 
-```
+```bash
 $ ./configure --prefix=/opt/libfabric --disable-sockets && make -j 32 && sudo make install
```

Comment:
You removed `$` all but here.

Reply:
These were the original changes, which were not merged.  The merged changes keep the $ in all locations.